app/modals/reset-wallet/reset-wallet-modal.tsx

```diff
@@ -31,7 +31,7 @@ export const ResetWalletModal: FC<ResetWalletModalProps> = ({ isOpen, onClose })
     cancelBtnRef.current?.focus();
     timer.current = window.setTimeout(() => {
       void clearDiskStorage();
-      void api.reloadApp();
+      void main.reloadApp();
```

Comment:
we we want to reload app even if `clearDiskStorage` might still have a pending promise?

Reply:
We could mostly get away with this, as it's a separate process handling the delete (of the file on disk). Nonetheless you're right, there could be a race condition here.